### PR TITLE
feat(astro-angular): add ability to pass vite options to integration

### PR DESCRIPTION
**Pull request by marcjulian**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [x] vite-angular-plugin
- [x] astro-angular
- [ ] create-analog

## What is the current behavior?

Issue Number: #129

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by marcjulian** [comment]
@brandonroberts this is ready for review.

File: packages/astro-angular/README.md [hunk 1]
```diff
@@ -94,6 +94,31 @@ export default defineConfig({
...
+        tsconfig: 'path/to/tsconfig.app.json',
+        workspaceRoot: 'rootDir',
+        inlineStylesExtension: 'scss|sass|less|styl|stylus'
+      },
+    }),
+  ],
+});
+```
+
+The Astro Angular integration
```
Comment (brandonroberts) [inline packages/astro-angular/README.md:120]: Is this supposed to be here?

**Review by brandonroberts — APPROVED** [review]

File: packages/astro-angular/README.md (hunk 1 above)
Comment (marcjulian) [reply packages/astro-angular/README.md]: No I forgot about cleaning up.

**Comment by brandonroberts** [comment]
@allcontributors add @marcjulian for code

**Pull request merged**